Meta/src/meta.cpp

```diff
@@ -158,6 +160,19 @@ void configureSourceVariableThreadPool(const Configuration::SourceVariableThread
     MetaUtils::linkHandlerObjectAndAddressSpaceNode(dSourceVariableThreadPool, asSourceVariableThreadPool);
 }
 
+void configureBuildInformation(const Configuration::BuildInformation& config, AddressSpace::ASNodeManager *nm,  AddressSpace::ASStandardMetaData* parent)
```

Comment:
Same question here as few questions above - I don't understand why this purely runtime information is routed via Configuration objects.

Reply:
and same answer (yep - it's a fair point)

Reply:
Done